### DISCUSSION TITLE
Cannot see in UI to which notification channels/policies an alert is linked

### DISCUSSION THREAD

**Discussion by BlackDex** [grafana/grafana#57360] [Alerting]
Almost the same as #52677, but besides not being able to see if an alert is silenced it would also be very useful to see to which channels a notification would be sent. Now you have guess or search multiple labels to find a match.

This makes it very difficult to check if a specific alert is sent to the correct channels.
Especially when there are multiple teams which all have there own notification channels and maybe some shared.

Having it visible in the same location as the alert it self for a quick overview.

**Environment**:
- Grafana version: v9.2.0 (Unified Alerting enabled)
- Data source type & version: n/a
- OS Grafana is installed on: Enterprise Docker Image
- User OS & Browser: Doesn't matter
- Grafana plugins: n/a
- Others: n/a

**Comment by ashharrison90** [comment]
hey @BlackDex 👋 

thanks for raising this! 🙌  agreed, this would be a valuable feature to have. 

our process for feature requests now is to use a github discussion to gauge feedback from the team and community (see the faq [here](https://github.com/grafana/grafana/discussions/36962)). once the team is happy to commit to implementing the feature, they'll convert it back to an issue and start working on it. 

hope that makes sense 🙌

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
